### DISCUSSION TITLE
[Show and Tell] <Rivo.xyz - DeFi Aggregator combining AA wallet & DeFi marketplace>

### DISCUSSION THREAD

**Discussion by vseriousv** [ethereum-optimism/developers#164] [Show and tell]
### Description

[Rivo.xyz](https://rivo.xyz/) is a DeFi platform that combines a multichain smart-contract wallet with a DeFi marketplace, unlocking seamless, non-custodial access to cross-chain yield opportunities. 

We've released Rivo Beta on Optimism L2 blockchain in November 20 with two available DeFi strategies: 

- [Yearn OP](https://app.rivo.xyz/asset/0x7D2382b1f8Af621229d33464340541Db362B4907), a vault created by Yearn Finance that generates yield through the lending of OP tokens on Sonne Finance and acquiring additional rewards in SONNE tokens.
- [Yearn USDC](https://app.rivo.xyz/asset/0xaD17A225074191d5c8a37B50FdA1AE278a2EE6A2), a vault created by Yearn Finance that generates yield by supplying USDC on the Aave lending market and acquiring additional rewards in OP tokens.

## Here's some of our key features on Optimism:

- Seedless web3 wallet creation with Gmail and socials 
- One-click entry and exiting in DeFi strategies
- Asset page with exclusive content: charts, dashboards, short videos, risk assessments and audits
- Fiat-to-crypto onramp 

## Beta version metrics

- 2 vaults on Optimism 
- 1000+ connected EOA wallets 
- 300+ created Account Abstraction wallets 
- $100,000+ total platform volume since Beta launch
- 1000+ swaps worth over $50k 

## Community
If you want to connect, ask any questions, join our [Discord](https://discord.gg/rivoxyz) server or follow updates on [Twitter](https://twitter.com/rivoxyz).

**Comment by sbvegan** [comment]
Very cool, thanks for sharing!

**Reply by sbvegan** [reply]
If you're interested in connecting with OP Labs on what you're building, please fill out this [form](https://airtable.com/appQ2S9IatldkoNQw/pagyJSUyxJA2SQKO3/form).
